### ISSUE TITLE
Set up server

### ISSUE THREAD

**Issue by sofiapoh** [pursuit-of-hapiness/wild-wild-wordpress#11]

**Comment by matthewiiv** [comment]
Endpoints:

```
/content
'GET' - read content
'POST' - create new content
'PUT' - update content
'DELETE' - remove content

/user
'POST' - Create User
'POST' ?q=login&username=un&password=pw
```

**Comment by matthewiiv** [comment]
Content routes done

**Comment by mcelearr** [comment]
Fixed by #27
